src/tests/ftest/telemetry/wal_metrics.py

```diff
@@ -98,8 +98,8 @@ def test_wal_reply_metrics(self):
                         # Replay size should be > 0 after pool create for MD on SSD
                         ranges[metric][label] = [1]
                     elif metric.endswith('_replay_time'):
-                        # Replay time should be 10,000 - 50,000 after pool create for MD on SSD
-                        ranges[metric][label] = [10000, 50000]
+                        # Replay time should be 5,000 - 50,000 after pool create for MD on SSD
+                        ranges[metric][label] = [5000, 50000]
```

Comment:
In the extreme case when everything is checkpoint-ed before replay, the replay will only load single block, then call an unmap. Maybe we could adjust it to an even smaller value, like 10 (us)?